blaze-core/src/main/scala/org/http4s/blazecore/websocket/Http4sWSStage.scala

```diff
@@ -23,6 +24,8 @@ private[http4s] class Http4sWSStage[F[_]](
 )(implicit F: ConcurrentEffect[F], val ec: ExecutionContext)
     extends TailStage[WebSocketFrame] {
 
+  private[this] val writeSemaphore = F.toIO(Semaphore[F](1L)).unsafeRunSync()
```

Comment:
This is problematic but solves the problem. :cry:

Reply:
Hate breaking RT

Reply:
I don't like it, but `new Http4sWSStage(...)` is already not RT, and this value can't be observed outside the instance.  This is at the far end of town where the grickle grass grows and the wind smells slow and sour when it blows.